src/frontend/shortlist/src/App.vue

```diff
@@ -10,6 +10,22 @@ const apiClient = new ShortlistApi("https://api.shortlist.nyc");
 const router = useRouter();
 const appSessionStore = sessionStore();
 
+function moveAcceptedSchoolCardToDB(payload) {
```

Comment:
Can this be markSchoolAsAccepted instead?
```suggestion
function markSchoolAsAccepted(payload) {
```